packages/node_modules/@webex/internal-plugin-devices/src/devices.js

```diff
@@ -320,7 +328,28 @@ const Devices = WebexPlugin.extend({
 
   // Ampersand method members
 
-  initialize() {} // Sets up the plugin as well as its events.
+  /**
+   * Initializer method for the devices plugin.
+   *
+   * @override
+   * @param {Array<any>} args - An array of items to be mapped as properties.
+   * @returns {void}
+   */
+  initialize(...args) {
```

Comment:
I added the FeaturesModel event triggers here, now that features are implemented.